kt/godot-library/src/main/kotlin/godot/signals/SignalProvider.kt

```diff
@@ -3,38 +3,6 @@ package godot.signals
 import godot.Object
 import kotlin.reflect.KProperty
 
-/**
- * For java convenience usage
- */
-object SignalProvider {

```

Comment:
I don't see replacements for all those methods in the generated code. If I recall they exist because Java can't use the same syntax as Kotlin `val signal by signal1<Int>` so Cedric created those to help.